Cisco/iOS-XE/snmp.conf

```diff
@@ -0,0 +1,9 @@
+! SNMP IOS-XE
+!
+! ACL for SNMP Access
+access-list 33 remark SNMP Query ACL
+access-list 33 permit 209.50.158.0 0.0.1.255
+!
+! Sets Access and the community
+snmp-server community <SNMP_COMMUNITY> RO 33
```

Comment:
In this line, using `{{snmp_community}}` will leverage templating and auto-replace with what is inserted in the portal UI at device provisioning time - this isn't leveraged in the current UI, but will be in the forthcoming device provisioning UI which will suggest configuration snippets directly related to the values entered in the device provisioning web form. (the Juniper templates already leverage that `{{}}` notation.

Reply:
yes!